src/contexts/game.context.tsx

```diff
@@ -16,6 +34,15 @@ type GameProps = { children?: ReactNode };
 
 export const GameProvider: React.FC<GameProps> = ({ children }) => {
     const [isTriggered, setIsTriggered] = useState(false);
+    const [story,setStory] = useState<Story>({entries: [], opener:''});
+    const [content, setContent] = useState('');
+    const addEntry = useCallback((storyEntry: StoryEntry) => {
+        setStory({...story, entries: [...story.entries, storyEntry]});
```

Comment:
As we talked about, lets please have single object name gameData to store all game's dynamic data.